docs/customization/suites.rst

```diff
@@ -0,0 +1,257 @@
+                                default_sales_tax:
+                                    code: "sales_tax_20"
+                                    name: "Sales Tax 20%"
+                                    zone: "PL"
+                                    category: "other"
+                                    amount: 0.2
+
+                    promotion: # add your promotion
+                        options:
+                            custom:
+                                black_friday: # promotion with primary settings
+                                    code: "black_friday"
+                                    name: "Black Friday"
+                                    channels:
+                                        - "PL_WEB"
+                                new_year: # promotion with more settings
+                                    code: "new_year"
+                                    name: "New Year"
+                                    usage_limit: 10
+                                    priority: 2
+                                    starts_at: "-7 day"
+                                    ends_at: "7 day"
+                                    channels:
+                                        - "PL_WEB"
+                                    rules:
+                                        -
+                                            type: "item_total"
+                                            configuration:
+                                                PL_WEB:
+                                                    amount: 100.00
+                                    actions:
+                                        -
+                                            type: "order_fixed_discount"
+                                            configuration:
+                                                PL_WEB:
+                                                    amount: 10.00
+
+                    # add your products:
+                    mug_product:
+                        options:
+                            amount: 30
+
+                    sticker_product:
+                        options:
+                            amount: 20
+
+                    book_product:
+                        options:
+                            amount: 15
+
+                    tshirt_product:
+                        options:
+                            amount: 15
+
+                    product_review:
+                        options:
+                            random: 50
+
+                    similar_product_association:
+                        options:
+                            amount: 20
+
+                    order:
+                        options:
+                            amount: 20
+
+                    address:
+                        options:
+                            random: 10 # the number of addresses that are random created
+                            prototype:
+                                country_code: PL
+
```

Comment:
I'm missing where we are telling the user to call `bin/console sylius:fixtures:load poland` explicitly